test/domains/neth/nonHomoScalar1.good

```diff
@@ -0,0 +1 @@
+nonHomoScalar1.chpl:1: error: Tuple bounds must be homogenous. Got (int(64),real(64))
```

Comment:
I suspect you want to say "tuples defining rectangular domain bounds must be homogeneous" or something, otherwise it's not clear how this relates to the domain at all.

Reply:
Your other error message for this type of error is much clearer and I think you should re-use it here.